src/test/scala/examples/RiscTests.scala

```diff
@@ -47,7 +47,8 @@ class RiscTests(c: Risc) extends PeekPokeTester(c) {
 
 class RiscTester extends ChiselFlatSpec {
   behavior of "Risc"
-  backends foreach {backend =>
+//  backends foreach {backend =>
+  Seq("firrtl") foreach {backend =>
```

Comment:
Are we dropping backends intentionally or is this a scalpel left in the patient?

Reply:
Ouch!